### DISCUSSION TITLE
Cant go back from solana Wormhole to Eth Mainnet  (Wbtc)

### DISCUSSION THREAD

**Discussion by Lur50** [wormhole-foundation/wormhole#417] [Developer Q&A] [answered]
I transfer Wbtc from eth to solana via wormhole, and i can see it in my wallet un solana but when i want to send it back to eth via wormhole it said error. 

This is the trx

https://explorer.solana.com/tx/2vdRiDBfPw1oXFGJWUeAEbbt2xKbinXetFFUHJKo9H2Y66tYHmWbm12YruqYKaHHkKfYeps9PxFF4tDDFxx5MJhQ

i need help!
it is a whole btc stuck

**Comment by leoluk** [comment, marked as answer]
Make sure to have at least 0.05 SOL in your fee payer wallet

**Reply by Lur50** [reply]
Thanks it was the gas...
